src/vivarium_public_health/metrics/columns.py

```diff
@@ -0,0 +1,24 @@
+from typing import NamedTuple
+
+from vivarium.framework.results import VALUE_COLUMN
+
+
+class __Columns(NamedTuple):
```

Comment:
This container was copied/pasted from the old/deleted `reporters.py` (once I removed the now-not-used `write_results` method all that was left was this object and figured it made more sense here.